README.md

```diff
@@ -1,2 +1,38 @@
 # django-multiple-forms
 An expansion of django's FormView to handle multiple form submissions
+
+## How To Use
+#### Install the dependency
+```
+pip install django-mulitform-views
+```
+or add `django-multiform-views` to your `requirements.in` file, then run
+```
+pip compile
+pip install -r requirements.txt
+```
+
+#### Usage
+Have your view inherit from ```FormsView```.
+
+Append `multiform_key` to `helper.layout` in the form
+
+Add the fields for `template_name`, `success_url`, etc as you would normally for `FormView`
+
+Add the `form_classes` for each form you want in your view
+```
+form_classes = {
+    'first_form': FirstForm,
+    'second_form': SecondForm,
+}
+```
+The package will detect which form was submitted in the view. You can perform 
```

Comment:
Add: all form instantiation and handling methods from `FormView` can be defined individually for each form using the form key defined in `form_classes` as an infix, e.g. `get_initial` becomes `get_first_form_initial` and `get_form_kwargs` becomes `get_second_form_kwargs`.